openmc/deplete/results.py

```diff
@@ -512,10 +512,11 @@ def get_step_where(
         if math.isclose(time, times[ix], rel_tol=rtol, abs_tol=atol):
             return ix
 
+        closest = min(times, key=lambda t: abs(time - t))
```

Comment:
I guess we know that one of these three `times[ix - 1]`, `times[ix]`, or `times[ix + 1]` are close to what was requested. So we could look at those three points instead of the whole `times` array and same some time (pun intended)

Reply:
This would certainly save some time and I admit I had not thought of that. So I'm keen on this improvment.

If a user has just two entries in the timesteps then I guess it wouldn't be able to look at three points.